spec/views/welcome/about.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
The `spec/views` directory can be removed. View specs can be used to test the HTML rendered by a view template.

However, **feature specs** (which we'll walkthrough writing soon) are usually preferable to view specs for testing the rendered HTML. And added bonus about feature specs is that you can test what happens when a user fills in an HTML form, clicks a button, clicks a link, and anything else you can think of that a user would do in their browser when using your app.